jax/interpreters/pxla.py

```diff
@@ -402,7 +402,7 @@ def __getitem__(self, idx):
 ### the xla_pmap primitive and its rules are comparable to xla_call in xla.py
 
 def xla_pmap_impl(fun, *args, backend, axis_name, axis_size, global_axis_size,
-                  devices, name):
+                  devices, name, mapped_invars=None):
```

Comment:
This isn't a high priority, and we can ignore it for now, but I think it might be better to avoid default argument values. That is, we could just make sure `mapped_invars` is always provided as a parameter to `xla_pmap`. I think conventions like that can make debugging easier.